### PR TITLE
updated code for Unum instance

### PR DESCRIPTION
**Pull request by BhavanaR28**
INT-93 adding all adapters to Unum Environment

### CODE REVIEW

File: utils/apiFunctional.cjs [hunk 1]
```diff
@@ -68,7 +69,8 @@ const trigger = function(gatewayUrl, username, password, team_name, project_name
         method: 'POST',
         headers: {
             'Content-Type': 'application/json'
-        }
+        },
+        rejectUnauthorized: false
```
Comment (SPrasadT) [inline utils/apiFunctional.cjs:73]: Why are we using this flag?

File: utils/apiFunctional.cjs [hunk 2]
```diff
@@ -108,16 +109,19 @@ const trigger = function(gatewayUrl, username, password, team_name, project_name
...
     let apiCallConfig = {
         host: host_name,
         port: port,
         path: baseContext+'/checkExecutionStatus',
         method: 'POST',
         headers: {
             'Content-Type': 'application/json'
-        }
+        },
+        rejectUnauthorized: false
```
Comment (SPrasadT) [inline utils/apiFunctional.cjs:121]: why is this flag used?

File: utils/apiPerformance.cjs [hunk 3]
```diff
@@ -6,7 +6,7 @@ const path = require('path')
 const url = require('url')
 const { exec } = require("child_process");
 
-let baseContext = '/cli-adapter-api-testing/v1';
+let baseContext = '/cli-adapter-api-performance/v1';
```
Comment (SPrasadT) [inline utils/apiPerformance.cjs:9]: why there is a change in the context path? wasn't it the same from before or is it a new change?

File: utils/apiPerformance.cjs [hunk 4]
```diff
@@ -78,7 +78,8 @@ const trigger = function(gatewayUrl, username, password, team_name, project_name
         method: 'POST',
         headers: {
             'Content-Type': 'application/json'
-        }
+        },
+        rejectUnauthorized: false
```
Comment (SPrasadT) [inline utils/apiPerformance.cjs:82]: why is this flag used?

File: utils/web.cjs [hunk 5]
```diff
@@ -7,7 +7,7 @@ let count1 = 0;
 let oldSet = new Set();
 let newSet = new Set();
 let array3 =[];
-let baseContext = '/cli-adapter-web-repository/v1';
+let baseContext = '/cli-adapter-webrepository/v1';
```
Comment (SPrasadT) [inline utils/web.cjs:10]: why is there is a change in this? how was this working before?

**Review by SPrasadT — CHANGES_REQUESTED** [review]

File: utils/apiFunctional.cjs (hunk 1 above)
Comment (BhavanaR28) [reply utils/apiFunctional.cjs:73]: was getting self signed certificate in certificate chain error when ever we try to trigger Unum instance , so we used it

File: utils/apiFunctional.cjs (hunk 2 above)
Comment (BhavanaR28) [reply utils/apiFunctional.cjs:121]: was getting self signed certificate in certificate chain error when ever we try to trigger Unum instance , so we used it

File: utils/apiPerformance.cjs (hunk 3 above)
Comment (BhavanaR28) [reply utils/apiPerformance.cjs:9]: it is new change

File: utils/apiPerformance.cjs (hunk 4 above)
Comment (BhavanaR28) [reply utils/apiPerformance.cjs:82]: was getting self signed certificate in certificate chain error when ever we try to trigger Unum instance , so we used it

**Pull request closed without merging**